fix: combine enable switch for basic monitoring and prober monitoring (#262)
abcxyz/jvs: docs/prober-and-monitoring.md

}
```

> By default, alering is disabled, you can enable alerting for cloud run services and prober by setting the following variables:

Reviewer: ```suggestion
By default, alering is disabled, you can enable it by setting the following variables:
```
